utils/constants.go

```diff
@@ -102,6 +102,8 @@ const (
 	LeaderService ResourceName = "leader-service"
 	// FollowerService is the name of a service that points healthy followers (excludes leader).
 	FollowerService ResourceName = "follower-service"
+	// MetricsService is the name of the service that points to all nodes.
```

Comment:
comment information is uncompleted, ` MetricsService is the name of the metrics service that points to all nodes.` 
may be good